.circleci/local_publish_helpers.sh

```diff
@@ -31,7 +31,15 @@ function uploadPkgCli {
         aws --profile=s3-uploader s3 cp amplify-pkg-linux-arm64.tgz s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-arm64-$(echo $hash).tgz
         aws --profile=s3-uploader s3 cp amplify-pkg-linux-x64.tgz s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-x64-$(echo $hash).tgz
 
-        if [ "0" -ne "$(aws s3 ls s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-x64 | egrep -v "amplify-pkg-linux-x64-.*" | wc -l)" ]; then
+        INCORRECT_FILES="$(set -o pipefail && aws --profile=s3-uploader s3 ls s3://aws-amplify-cli-do-not-delete/$(echo $version)/amplify-pkg-linux-x64 | ( egrep -v "amplify-pkg-linux-x64-.*" || true ) | wc -l | xargs)"
+        INCORRECT_PERMISSIONS=$?
+
+        if [ INCORRECT_PERMISSIONS -ne "0" ]; then
```

Comment:
I see below you tested with double bracketed conditions but here they are single brackets. My bash knowledge isn't good enough to know if it matters here but I know there are subtle differences between the two. Just making sure this was intentional